### ISSUE TITLE
Import error message when starting LibreCAD

### ISSUE THREAD

**Issue by lucasgamaleri** [LibreCAD/LibreCAD#1553]
### Expected behavior
Expected LibreCAD to open into a new file when starting.

### Observed behavior
When you start LibreCAD, it shows an Import error: error opening DXF/DWG file.

### Steps to reproduce or sample file
Open LibreCAD in default configuration, in my case I just installed from Sourceforge.
It's the first error message that appears

### Operating System and LibreCAD version info
Operating System:
Edition Windows 10 Home
Version	21H2
OS Compilation 19044.1806

LibreCAD version info:
Version: 2.2.0-rc3-24-g4602ec6a
Compiler: GNU GCC 7.3.0
Compiled on: Mar  5 2022
Qt Version: 5.12.4
Boost Version: 1.65.1
System: Windows 10 (10.0)
![image](https://user-images.githubusercontent.com/11754115/177560885-074f5755-1ae0-4a2a-aa45-dfdb2ac0370b.png)

**Comment by lordofbikes** [comment]
Works for me!
Can you detail your steps?
Any issues during installation?
Used administrator privileges anywhere?

**Comment by lordofbikes** [comment]
Had this recently too, but no chance to investigate.
But pretty sure, this is a Windows only issue.

**Comment by lucasgamaleri** [comment]
I agree it must be a OS specific issue. I couldn't build the project from source on VS nor Qt by following the repository wiki so couldn't investigate. I might keep trying in the future
